code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -128,6 +132,7 @@
 /obj/item/gun_upgrade/trigger/cop_block
 	name = "Frozen Star \"Cop Block\" Trigger"
 	desc = "A simpler way of making a weapon display-only"
+	matter = list(MATERIAL_PLASTEEL = 2)
```

Comment:
a trigger to make a gun be a dud could just cost steel or plastic. should be cheap

Reply:
It's a fairly rare find in maint and requires two minutes of standing still to remove, so it'd be logical for it to be made out of something tough